packages/shopify-codemod/test/fixtures/remove-empty-catch-blocks/basic.input.js

```diff
@@ -0,0 +1,17 @@
+try {
+  foo();
+} catch (undefined) {
+
+} finally {
+  bar();
+}
+
+try {
+  foo();
+} catch (e) {} finally {
```

Comment:
Please add a test case with a non-empty `catch` block to show they are not removed.